### ISSUE TITLE
Wrong documentation + incorrect implementation for `ping_timeout`?

### ISSUE THREAD

**Issue by mkonecny** [websocket-client/websocket-client#237]
In the code it says

> ping_timeout: timeout(second) **if the pong message is not received.**

But this doesn't appear to be true. The only thing `ping_timeout` is used for is to compare when the last PING was sent, and I don't see it related to PONG messages:

```
                if ping_timeout and self.last_ping_tm and time.time() - self.last_ping_tm > ping_timeout:
                    self.last_ping_tm = 0
                    raise WebSocketTimeoutException("ping timed out")
```

I see PONG messages only being used to call a user-defined callback:

```
                    elif op_code == ABNF.OPCODE_PONG:
                        self._callback(self.on_pong, frame.data)
```

**Comment by Nikhil13x** [comment]
Hello.. is this issue addressed?

**Comment by mkonecny** [comment]
Addressed in #241
